src/patterns/components/inputs/percentage.hbs

```diff
@@ -0,0 +1,34 @@
+---
+name: Percentage Input
+description:
+  This input is used to collect a percent value.
+information:
+  - Spark does not apply any validation or formatting to these fields.
+order: 23
+---
+
+<div class="sprk-b-InputContainer" data-sprk-input="percentage">
+  <div class="sprk-b-TextInputIconContainer sprk-b-TextInputIconContainer--has-text-icon-right">
```

Comment:
Hey @bobwise, this icon isn't a text icon anymore. Its a SVG so the class name of sprk-b-TextInputIconContainer--has-text-icon-right doesn't work. Needs some updating there.